.github/workflows/connect-webextension.test.yml

```diff
@@ -0,0 +1,24 @@
+name: connect-webextension e2e test
+
+on:
+  pull_request:
+    paths:
+      - "packages/connect-examples/**"
+      - "packages/connect-webextension/**"
```

Comment:
imho also 
packages/connect
packages/connect-iframe
packages/connect-web

could affect this. and also all internal subdependecies of these packages.